char led_color
bool led_on false
---
char r_led_color
bool r_led_on
